.github/workflows/cibuildwheel.yaml

```diff
@@ -148,14 +164,14 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           files: 'dist/*.tar.gz'
 
-      - name: Publish sdist to TestPyPI
+      - name: Publish sdist and pure-Python wheel to TestPyPI
```

Comment:
I think we don't download the pure python wheel.

It's likely fine to just upload that this the other wheels, so let's just revert the name change here and below

Reply:
We do actually download it in this step (I think). Let's wait until the Action passes until that point, the emulation is taking a fair bit of time. :)

Reply:
Oh sorry, I read `files: 'dist/*.tar.gz'` just above and though it was the download artifact action